packages/block-library/src/cover/edit.js

```diff
@@ -604,7 +618,7 @@ function CoverEdit( {
 	const classes = classnames(
 		dimRatioToClass( dimRatio ),
 		{
-			'is-dark-theme': isDark,
```

Comment:
I couldn't find any evidence that this was used so I figured it was ok to change the name.

Reply:
It used to be applied when a theme opted into "dark editor styles", which would apply that CSS class to the body element. 

It's used in a bunch of blocks to invert the block editor UI to be visible, such as changing the spacer block background gray to white:

<img width="451" alt="Screenshot 2021-08-02 at 11 07 09" src="https://user-images.githubusercontent.com/1204802/127835476-95d08375-851b-4e3f-9014-2a2651b581c9.png">

I think there's probably a better way to do this, it's a bit hacky for this block to apply the class so that nested blocks inside show inverted block UI. But I can't think of an easy solution to doing so.